srcpkgs/rustup/template

```diff
@@ -13,19 +14,25 @@ homepage="https://www.rustup.rs"
 distfiles="https://github.com/rust-lang/${pkgname}/archive/${version}.tar.gz"
 checksum=ad46cc624f318a9493aa62fc9612a450564fe20ba93c689e0ad856bff3c64c5b
 
+post_build() {
+	# rustup errors out because it doesn't know about armv6l-musl
+	if [ "$XBPS_TARGET_MACHINE" != armv6l-musl ]; then
```

Comment:
I *think* there are a bunch more than just armv6l-musl. Going off https://forge.rust-lang.org/release/platform-support.html:

- armv5tel
- armv5tel-musl
- armv7l-musl (??)
- aarch64-musl (??)
- i686-musl
- mips* (we don't have glibc variants for mips)

Reply:
So, the problem here is that `rustup` doesn't even know that `armv6l-musl` exists, so it errors out with a rather cryptic error. For the others, we might not have upstream binaries, but `rustup` knows about them and can work normally. I can try to build it for mips / armv5 to check those too.

Reply:
I wouldn't go too crazy here. If you try building for archs that don't have binary packages, you are going to go crazy building a bunch of stuff just to get to the point where you can test your package. The main idea should be to prevent issues with trying to build official binary packages. If somebody wants to try building rustup for mips, let the user with the problem get involved with resolving any problems that shake out.

Reply:
Fair enough, both good points